CHANGELOG.md

```diff
@@ -92,6 +92,7 @@ current changes on git with [previous release tags][git_tag_comparison].
 - [Added `set_minimized` and `set_position` to `Window`][1292]
 - [Example for 2D Frustum Culling][1503]
 - [Add remove resource to commands][1478]
+- [Add webp as a supported texture format (C compiler required)][2600]
```

Comment:
Thats the Changelog of the already released 0.5
0.6 does not have a Changelog yet.

Reply:
fixed.